bytecomp/bytegen.ml

```diff
@@ -345,11 +345,11 @@ let comp_primitive p args =
   | Pfloatcomp Cgt -> Kccall("caml_gt_float", 2)
   | Pfloatcomp Cle -> Kccall("caml_le_float", 2)
   | Pfloatcomp Cge -> Kccall("caml_ge_float", 2)
-  | Pstringlength -> Kccall("caml_ml_string_length", 1)
-  | Pstringrefs -> Kccall("caml_string_get", 2)
-  | Pstringsets -> Kccall("caml_string_set", 3)
-  | Pstringrefu -> Kgetstringchar
-  | Pstringsetu -> Ksetstringchar
+  | Pstringlength | Pbyteslength -> Kccall("caml_ml_string_length", 1)
```

Comment:
I don't think that bootstrapping is required any more when primitives are added.

I'm concerned with the system be left in an inconsistent state, with the runtime system making the distinction between string and bytes primitives only for some operations.  I'm fine with the current PR if @hhugo (or someone else) is indeed willing to finish the transition.